src/game/puppyprint.c

```diff
@@ -438,9 +435,10 @@ void append_puppyprint_log(const char *str, ...) {
     osSyncPrintf(textBytes);
 #endif
     for (u8 i = 0; i < (LOG_BUFFER_SIZE - 1); i++) {
-        memcpy(consoleLogTable[i], consoleLogTable[i + 1], 255);
+        memcpy(consoleLogTable[i], consoleLogTable[i + 1], sizeof(consoleLogTable[0]));
     }
-    memcpy(consoleLogTable[LOG_BUFFER_SIZE - 1], textBytes, 255);
+    memcpy(consoleLogTable[LOG_BUFFER_SIZE - 1], textBytes, sizeof(consoleLogTable[0]));
```

Comment:
Nit: use `LOG_MESSAGE_SIZE` instead of `sizeof`

Reply:
Done